doc/main/tbb_userguide/Guiding_Task_Scheduler_Execution.rst

```diff
@@ -44,19 +44,24 @@ assign a NUMA node identifier to the ``task_arena::constraints::numa_id`` field.
 
 ::
 
-    std::vector<tbb::numa_node_id> numa_indexes = tbb::info::numa_nodes();
-    std::vector<tbb::task_arena> arenas(numa_indexes.size());
-    std::vector<tbb::task_group> task_groups(numa_indexes.size());
-
-    for(unsigned j = 0; j < numa_indexes.size(); j++) {
-        arenas[j].initialize(tbb::task_arena::constraints(numa_indexes[j]));
-        arenas[j].execute([&task_groups, &j](){ 
-            task_groups[j].run([](){/*some parallel stuff*/});
-        });
+    std::vector<tbb::numa_node_id> numa_nodes = tbb::info::numa_nodes();
```

Comment:
Can we also separate this example into the .cpp file in `examples` folder to make it testable?